Doc/distutils/sourcedist.rst

```diff
@@ -95,8 +95,9 @@ source distribution:
   distributions, but in the future there will be a standard for testing Python
   module distributions)
 
-* :file:`README.txt` (or :file:`README`), :file:`setup.py` (or whatever  you
-  called your setup script), and :file:`setup.cfg`
+* Any of the standard README files (:file:`README`, :file:`README.txt`,
+  or :file:`README.rst`), :file:`setup.py` (or whatever you called your setup
+  script), and :file:`setup.cfg`.
```

Comment:
You should document the change using a ".. versionchanged:: 3.7 xx" markup, see a few lines below.